templates/addons/cilium/cilium.yaml

```diff
@@ -6,6 +6,7 @@ spec:
   clusterSelector:
     matchExpressions:
       - {key: ipv6, operator: DoesNotExist}
+      - {key: vxlan, operator: DoesNotExist}
```

Comment:
I think we might also need this on the cilium-ipv6.yaml?

Reply:
By adding it to cilium-ipv6, are you trying to say we need to protect from cases where someone specifies something like:
```
    cni: cilium
    ipv6: "true"
    vxlan: "true"
```
I wonder if someone would do this. Let me know if you still think we should add such checks to addons.

Reply:
yeah we want to make sure only one cilium chart gets applied on a workload cluster, with the above labels I think both cilium-ipv6 and cilium-vxlan would try to be installed by CAAPH onto the workload cluster.

Reply:
sounds good, updated.